### ISSUE TITLE
Still traing phase shows epoch 1/30

### ISSUE THREAD

**Issue by imtinan39** [matterport/Mask_RCNN#2099]
Its been 3 hours it showing epoch 1/30.
My laptop has no gpu. Now what should i do?

**Comment by Ankin01** [comment]
Try using a GPU for training. Search for online available GPUs that you can use.

**Comment by imtinan39** [comment]
Tried! But this time not utilizing GPU.
tensorflow-1.5
Tensorflow-1.5
Keras-2.3.1 
These are the version i am Currently using.

**Comment by Ankin01** [comment]
Tensorflow-gpu?

**Comment by imtinan39** [comment]
Yes! On colab  training start but using only cpu!

**Comment by Ankin01** [comment]
As per my knowledge Google Colab has two versions of Tensorflow installed by default <1.15.0 or 2.2.0 with which you can switch in between. Please check https://colab.research.google.com/notebooks/tensorflow_version.ipynb both of which are not currently supported by default in Mask RCNN library. 
If you have installed Tensorflow using pip (which is not recommended on Colab-check link above for this- and you are sure its working correctly) you may try to change the runtime environment from cpu to gpu that is 
Runtime -> Change Runtime Type -> Hardware Accelerator -> GPU. 
To check whether your Tensorflow is working correctly please check the version.
